src/sdk/models/metadataFilters.ts

```diff
@@ -0,0 +1,81 @@
+export enum MetadataConjunctionOperator {
+	And = 'and',
+	Or = 'or'
+}
+
+export enum MetadataPropertyFilterOperator {
+	eq = 'Equals',
+	ne = 'Not Equal',
+	lt = 'Less Than',
+	le = 'Less Than Equal To',
+	gt = 'Greater Than',
+	ge = 'Greater Than Equal To',
+	starts_with = 'Starts With',
+	not_starts_with = 'Does Not Start With',
+	ends_with = 'Ends With',
+	not_ends_with = 'Does Not End With',
+	contains = 'Contains',
+	not_contains = 'Does Not Contain',
+	in = 'In',
+	not_in = 'Not In',
+	empty = 'Empty',
+	not_empty = 'Not Empty',
+}
+
+export enum MetadataFunctionName {
+	Now = 'Now',
+	Year = 'Year',
+	Month = 'Month',
+	Day = 'Day',
+	AddDays = 'Add Days',
```

Comment:
The part in quotes should match the function name, which shouldn't have any spaces in it.

Reply:
Updated